src/modules/Builder/components/PackageList/components/PackagesTable/styles.scss

```diff
@@ -2,3 +2,8 @@
     margin: 0 5px;
     display: inline-block;
 }
+@media screen and (max-width:576px){
+    .adaptive table thead tr #Description, .adaptive table tbody tr #description{
```

Comment:
1. Here you also need to replace with `packages-table-card`. 
2. Just specify `.packages-table-card #Description` and `.packages-table-card #description`.